example/nodejs/Dockerfile

```diff
@@ -1,24 +1,22 @@
 ARG DOCKER_PROXY_HOST="docker.io"
 FROM ${DOCKER_PROXY_HOST}/node:16
 
-RUN mkdir -p /example/nodejs
-RUN chown -R root:root /example/nodejs
+USER node
 ENV HOME /example/nodejs
-USER root
 WORKDIR $HOME
 
 # Copy source files
-COPY package.json ${HOME}/package.json
-COPY src ${HOME}/src
-COPY tsconfig.json ${HOME}/tsconfig.json
-COPY tsconfig.production.json ${HOME}/tsconfig.production.json
-COPY cert ${HOME}/cert
+COPY --chown=node:node package.json ${HOME}/package.json
+COPY --chown=node:node src ${HOME}/src
+COPY --chown=node:node tsconfig.json ${HOME}/tsconfig.json
+COPY --chown=node:node tsconfig.production.json ${HOME}/tsconfig.production.json
+COPY --chown=node:node cert ${HOME}/cert
 
 # Setup project
-RUN npm install --no-progress && npm cache verify
```

Comment:
when i build docker image locally then build is failed due to failed to resolve dependencies, so i added `--legacy-peer-deps`.
full error log is bellow:
```
#0 27.31 npm ERR! code ERESOLVE
#0 27.33 npm ERR! ERESOLVE unable to resolve dependency tree
#0 27.33 npm ERR! 
#0 27.33 npm ERR! While resolving: @viron/example-nodejs@1.0.0
#0 27.33 npm ERR! Found: typescript@5.3.3
#0 27.33 npm ERR! node_modules/typescript
#0 27.33 npm ERR!   dev typescript@"^5.1.6" from the root project
#0 27.33 npm ERR! 
#0 27.33 npm ERR! Could not resolve dependency:
#0 27.33 npm ERR! peer typescript@"^4.0.0" from eslint-config-cam-recommended@1.1.1
#0 27.33 npm ERR! node_modules/eslint-config-cam-recommended
#0 27.33 npm ERR!   dev eslint-config-cam-recommended@"^1.1.1" from the root project
#0 27.33 npm ERR! 
#0 27.33 npm ERR! Fix the upstream dependency conflict, or retry
#0 27.33 npm ERR! this command with --force, or --legacy-peer-deps
#0 27.33 npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
#0 27.33 npm ERR! 
#0 27.33 npm ERR! See /example/nodejs/.npm/eresolve-report.txt for a full report.
#0 27.35 
#0 27.35 npm ERR! A complete log of this run can be found in:
#0 27.35 npm ERR!     /example/nodejs/.npm/_logs/2024-01-08T15_50_36_372Z-debug-0.log
```